src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
+            logger.info("Password check succesfull returning user...");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can`t login with user: {}", e.getMessage());
```

Comment:
just pass the exception, not the message
```suggestion
            logger.error("Can`t login with user: {}", e);
```

Reply:
Actually we do not need those braces too, my bad, did not see that at the first look
we need them when we want to add something to the message like, but the exceptions can be passed without it